connectors/gnosis.ts

```diff
@@ -8,6 +8,9 @@ export default class Connector extends LockConnector {
   async connect() {
     let provider;
     try {
+      if (window?.parent === window) {
```

Comment:
@bonustrack We will still need this incase some already has the gnosis-safe connector saved inside their localStorage.